scripts/run_multi_model.py

```
@@ -0,0 +1,372 @@
+        "--tb=short",
+        "--timeout=120",
+        f"--junitxml={junit_xml_path}",
+    ]
+
+    if run_metadata and "start_time" in run_metadata and "end_time" in run_metadata:
+        start = run_metadata["start_time"]
+        end = run_metadata["end_time"]
+        duration = run_metadata.get("duration", (end - start).total_seconds())
+    else:
+        start = datetime.now(timezone.utc)
+        result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)